src/markov/markov_approx.jl

```diff
@@ -219,3 +223,567 @@ function estimate_mc_discrete{T}(X::Vector{T})
 
     return estimate_mc_discrete(X, states)
 end
+
+"""
+
+types specifying the method for `discrete_var`
+
+"""
+@compat abstract type AbstractMethod end
+immutable Even <: AbstractMethod end
+# immutable type Quantile <: AbstractMethod end
+# immutable type Quadrature <: AbstractMethod end
+
+doc"""
+
+Compute a finite-state Markov chain approximation to a VAR(1) process of the form
+
+```math
+    y_{t+1} = b + By_{t} + \Psi^{\frac{1}{2}}\epsilon_{t+1}
+```
+
+where ``\epsilon_{t+1}`` is an (M x 1) vector of independent standard normal
+innovations
+
+```julia
+P, X = discrete_var(b, B, Psi, Nm, nMoments, method, nSigmas)
+```
+
+##### Arguments
+
+- `b::ScalarOrArray` : (M x 1) constant vector
+                       (M=1 corresponds scalar case)
+- `B::ScalarOrArray` : (M x M) matrix of impact coefficients
+- `Psi::ScalarOrArray` : (M x M) variance-covariance matrix of the innovations
+- `Nm::Integer > 3` : Desired number of discrete points in each dimension
+
+##### Optional
+
+- `nMoments::Integer` : Desired number of moments to match. The default is 2.
+- `method::AbstractMethod` : AbstractMethod specifying the method used to determine
+              the grid points. Accepted inputs are `Even()`. Please see the
+             paper for more details.
+             NOTE: `Quantile()` and `Quadrature()` are not supported now.
+- `nSigmas::Real` : If the `Even()` option is specified, nSigmas is used to
+              determine the number of unconditional standard deviations
+              used to set the endpoints of the grid. The default is
+              sqrt(Nm-1).
+
+##### Returns
+
+- `P` : (Nm^M x Nm^M) probability transition matrix. Each row
```

Comment:
@sglyon Since math-rendered x looks differently, I avoid math backticks.

I added code-backticks to make it consistent with other existing docs.